spec/README-compat.md

```diff
@@ -0,0 +1,139 @@
+
+# Wasm Image Specification v0.0.0
+
+## Introduction:
+
+This document describes a varient of [Wasm Artifact Image Specification](README.md), which leverages the standard layer media types. Here, we omit definition and terminology explained in [Wasm Artifact Image Specification](README.md). 
+
+We call this variant "compat", and the spec in [Wasm Artifact Image Specification](README.md) "oci".
+
+## Description
+
+This *compat* variant makes use of standard media type for layers, and is not based on custom OCI Artifcat media types. This way users can oeperate with standard tools such as docker, podman, buildah, etc.
+
+## Specification
+
+### Layer
+
+The *compat* variant must consist of exactly one layer whose media type is one of the followings:
+- `application/vnd.oci.image.layer.v1.tar+gzip`
+- `application/vnd.docker.image.rootfs.diff.tar.gzip`
+
+In addition, the layer must consist of the following two files:
+- `plugin.wasm` - (**Required**) A Wasm binary to be loaded by the runtime.
+- `runtime-config.json` - (**Optional**) A runtime configuratio specified in [Wasm Artifact Image Specification](README.md).
```

Comment:
Link to specific anchor is preferred showing runtime-config.json.

Reply:
Somewhere is should be clear that plugin.wasm and a json file is the common payload, and we have 3 ways of packaging them based on your needs.

Reply:
yeah agreed. Maybe better add a `DEVELOPER.md` or any separate markdown for explaining these three ways and how Istio will support them. sg?

Reply:
done. https://github.com/solo-io/wasm/pull/265/commits/ee52cb636f7f2fbf4054a8885efcac690def0b55